packages/devextreme/js/__internal/ui/chat/chat_avatar.ts

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import $ from '@js/core/renderer';
+import type { WidgetOptions } from '@js/ui/widget/ui.widget';
+
+import Widget from '../widget';
+
+const CHAT_MESSAGE_AVATAR_CLASS = 'dx-chat-message-avatar';
+const CHAT_MESSAGE_AVATAR_LETTERS_CLASS = 'dx-chat-message-avatar-letters';
+
+export interface AvatarOptions extends WidgetOptions<Avatar> {
+  name?: string;
+}
+
+class Avatar extends Widget<AvatarOptions> {
+  _getDefaultOptions(): AvatarOptions {
+    return {
```

Comment:
```suggestion
   return {
      ...super._getDefaultOptions(),
      name: '',
   };
```

Reply:
Check other places too

Reply:
In my code, there are four spaces there

Reply:
I only wanted to point out that you are using { name: '' } in multiple places. Spaces doesn't matter here.
'...{}' excess operation, I don't understand why we should leave it as is.